tests/mod.rs

```diff
@@ -74,28 +74,24 @@ fn assert_dirs_eq(expected: &Path, actual: &Path) {
 }
 
 fn run_test(name: &str) -> Result<(), cobalt::Error> {
-    let target = format!("tests/target/{}/", name);
-    let target: PathBuf = target.into();
-    let mut config = cobalt::ConfigBuilder::from_cwd(format!("tests/fixtures/{}", name))?;
-    let destdir = TempDir::new(name).expect("Tempdir not created");
-
-    config.source = "./".to_owned();
-    config.abs_dest = Some(destdir.path().to_owned());
+    let target = assert_fs::TempDir::new().unwrap();
+    target
+        .copy_from(format!("tests/fixtures/{}", name), &["*"])
+        .unwrap();
 
+    let mut config = cobalt::ConfigBuilder::from_cwd(target.path())?;
+    config.destination = "./_dest".into();
     let config = config.build()?;
     let destination = config.destination.clone();
-
-    // try to create the target directory, ignore errors
-    fs::create_dir_all(&destination).is_ok();
-
     let result = cobalt::build(config);
 
     if result.is_ok() {
-        assert_dirs_eq(&destination, &target);
+        let expected = format!("tests/target/{}", name);
+        let expected_path = Path::new(&expected);
+        assert_dirs_eq(&destination, expected_path);
```

Comment:
Replacing our custom dir-diffing is https://github.com/assert-rs/predicates-rs/issues/33